corehq/apps/domain/models.py

```diff
@@ -164,7 +166,16 @@ class InternalProperties(DocumentSchema, UpdatableSchema):
     goal_followup_rate = DecimalProperty()
     # intentionally different from and commtrack_enabled so that FMs can change
     commtrack_domain = BooleanProperty()
-    performance_threshold = IntegerProperty()
+    performance_threshold = IntegerProperty(default=15)
+    experienced_threshold = IntegerProperty(default=3)
```

Comment:
isn't this impossible to recover from once apps start getting saved with the defaults? (again i think fine if we're confident we won't change these but just pointing that out)